docs/tutorial/04_integrating_magma_access_gateway_with_magma_orchestrator.md

```diff
@@ -69,18 +69,17 @@ unit-magma-access-gateway-operator-0:
 ### Create a network in the `magma-test` organization:
 
 1. Login to the `magma-test` organization at this address: `https://magma-test.nms.<your domain>`. Use the credentials from the previous step.
-2. On the left pane, click on "Networks"
```

Comment:
Are we removing this line on purpose? I don't remember having any "Add Network" button on the main page.

Reply:
Yes, I was just installing it and when logging to the magma-test org, it put me directly on the network tab.

Reply:
Cool.